README.md

```diff
@@ -166,6 +166,18 @@ Given that you have included `spinkit.min.css` in your project, these snippets w
 </div>
 ```
 
+### Hexagon
+
+```html
+<div class="sk-hexagon">
+  <div class="sk-hex-parent">
```

Comment:
Could we get rid of this `sk-hex-parent` element? It's only used for positioning that could be applied directly to the children, right?